cli/azd/pkg/output/ux/created_repo_value.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package ux
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/azure/azure-dev/cli/azd/pkg/output"
+)
+
+type GitHubValueKind string
+
+const (
+	GitHubSecret   GitHubValueKind = "secret"
+	GitHubVariable GitHubValueKind = "variable"
+)
```

Comment:
We might only need a `bool` like `isSecret` instead of the enum.

Do you think there would be ever another kind here (other than value and sectet? )

Also, we should consider this change in a higher level and include Azdo.  I think we need to move the property to the pipeline manager and pipeline.  The pipeline manager should create the variables to be set for the pipeline and pass them to the pipeline implementation (github or azdo).  The variable should describe if the variable is a secret or not.. then the implementation would create the value depending on that

Reply:
I like that the `kind` is explicit and make it easier to generate the strings rather than have to do additional processing when writing out the message.

These types are only used in GitHub and this PR is only targeting GitHub right now.

Reply:
@vhvb1989 If there's something we need to do for azdo, please create an issue to track. If not currently needed, let's defer creation until we need to.

Reply:
well, I was trying to avoid re-doing the same thing later for azdo here. u.u

azdo logs also hide secrets, as we set the variables like secrets.  

It's fine to have this quick patch, for the rush,, but the solution should apply in general to any pipeline provider, as a part of the general interface abstraction...